exercises/pascals-triangle/test/test_pascals_triangle.c

```diff
@@ -10,25 +10,23 @@ void tearDown(void)
 {
 }
 
-static bool check(size_t count, size_t expected[][count], size_t ** result)
+static bool check_triangle(size_t count, size_t expected[][count],
+                           size_t ** actual)
 {
-   size_t i, j;
-   for (i = 0; i < count; i++) {
-      for (j = 0; j <= i; j++) {
-         if (expected[i][j] != result[i][j]) {
-            return 0;
-         }
-      }
-   }
-   return 1;
+   for (size_t i = 0; i < count; i++)
+      for (size_t j = 0; j <= i; j++)
+         if (expected[i][j] != actual[i][j])
+            return false;
+
+   return true;
 }
 
 static void test_no_rows(void)
 {
    size_t expected[1][1] = { {0} };
-   size_t **r = create_triangle(0);
```

Comment:
I wonder what `r` was supposed to mean. Result?

Reply:
Possibly. I wouldnt have made a change just for that but as I am working on the exercise anyway, and `actual` is what we've used everywhere else, it seemed like it would be clearer.